test/vagrant-service-manager/binary_handlers/adb_docker_binary_handler_test.rb

```diff
@@ -0,0 +1,100 @@
+require 'test_helper'
```

Comment:
do you mind making all the test_helper includes a `require_relative`. It helps me running tests in the IDE/debugger. Otherwise no difference really.